python/shotgun_model/shotgun_hierarchy_model.py

```diff
@@ -961,16 +938,22 @@ def hierarchy_is_supported(sg_connection):
 
     :param sg_connection: A shotgun connection.
 
-    :returns: ``True`` if hierarchy supported, ``False`` otherwise.
+    :returns: A tuple of 2 items where the first item is a boolean indicating
```

Comment:
I'm starting to use the `:rtype type:` and `:param type name: documentation`way of documenting stuff, I feel like the wording is better. I'm not sure this would help that much in this case, but I thought I'd mention it in case you wanted to experiment with it.